src/Sarif/NotYetAutoGenerated/ReportingDescriptor.cs

```diff
@@ -41,87 +41,88 @@ public SarifNodeKind SarifNodeKind
         public virtual MultiformatMessageString ShortDescription { get; set; }
 
         /// <summary>
         /// A description of the report. Should, as far as possible, provide details sufficient to enable resolution of any problem indicated by the result.
         /// </summary>
-        [DataMember(Name = "fullDescription", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "fullDescription", IsRequired = false, EmitDefaultValue = false, Order = 2)]
         public virtual MultiformatMessageString FullDescription { get; set; }
 
         /// <summary>
         /// A set of name/value pairs with arbitrary names. Each value is a multiformatMessageString object, which holds message strings in plain text and (optionally) Markdown format. The strings can include placeholders, which can be used to construct a message in combination with an arbitrary number of additional string arguments.
         /// </summary>
-        [DataMember(Name = "messageStrings", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "messageStrings", IsRequired = false, EmitDefaultValue = false, Order = 5)]
         public virtual IDictionary<string, MultiformatMessageString> MessageStrings { get; set; }
 
         /// <summary>
         /// Default reporting configuration information.
         /// </summary>
-        [DataMember(Name = "defaultConfiguration", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "defaultConfiguration", IsRequired = false, EmitDefaultValue = false, Order = 12)]
         public virtual ReportingConfiguration DefaultConfiguration { get; set; }
 
         /// <summary>
         /// A URI where the primary documentation for the report can be found.
         /// </summary>
-        [DataMember(Name = "helpUri", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "helpUri", IsRequired = false, EmitDefaultValue = false, Order = 3)]
         [JsonConverter(typeof(Microsoft.CodeAnalysis.Sarif.Readers.UriConverter))]
         public virtual Uri HelpUri { get; set; }
 
         /// <summary>
         /// Provides the primary documentation for the report, useful when there is no online documentation.
         /// </summary>
-        [DataMember(Name = "help", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "help", IsRequired = false, EmitDefaultValue = false, Order = 4)]
         public virtual MultiformatMessageString Help { get; set; }
 
         /// <summary>
         /// An array of objects that describe relationships between this reporting descriptor and others.
         /// </summary>
-        [DataMember(Name = "relationships", IsRequired = false, EmitDefaultValue = false)]
-        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        [DataMember(Name = "relationships", IsRequired = false, EmitDefaultValue = false, Order = 13)]
```

Comment:
>Order [](http://example.com/codeflow?start=90&length=5)

if JsonProperty tag exists we should also set it to the same because the default is -1 will get the filed at the first line.